jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
Why do you ask, is there a place where we output timestamps to the console? I took a quick look with this splunk query `index=testeng host="ip-10-0-4-206" source="/var/lib/jenkins/jobs/edx-platform-test-subset/*"` but offhand I don't see any.